src/sentry/conf/server.py

```diff
@@ -31,7 +31,6 @@
 from sentry.conf.types.service_options import ServiceOptions
 from sentry.conf.types.taskworker import ScheduleConfigMap
 from sentry.conf.types.uptime import UptimeRegionConfig
-from sentry.utils import json  # NOQA (used in getsentry config)
```

Comment:
I wouldn't remove this, it may still be used by getsentry config files.

Reply:
ah I must have fat fingered this